### PR TITLE
feat: write all log headers use case

### PR DESCRIPTION
**Pull request by paultrygs**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -41,15 +36,4 @@
...
-            extract_attributes = dataframe.loc[:, attributes]
-
-            # Finally, only include logs whose name contains "STAT"
-            filtered_log_headers = extract_attributes.loc[extract_attributes["NAME"].str.contains("STAT")]
-
-            # Print first three log entries:
-            print(filtered_log_headers.loc[0:3, :])
-
-            filtered_log_headers.to_csv(file, mode='a', index=False, header=False)
+    _ = write_log_headers_to_file(attributes=attributes, project=project, file_path="data/test.csv")
```
Comment (KristianKjerstad) [inline main.py:39]:
```suggestion
    write_log_headers_to_file(attributes=attributes, project=project, file_path="data/test.csv")
```

do we need _ = ?
looks like write_log_headers_to_file does not return anything

File: src/use_cases/logs.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+from src.repositories.get_repository import get_dsis_log_repository
+
+
+def write_log_headers_to_file(attributes: List[str],
+                              project: str,
+                              file_path: str,
+                              log_repository: LogInterface = get_dsis_log_repository()):
+    """
+    GET selected log attributes from all logs at input project and
+    write the data to input file path as csv file.
```
Comment (KristianKjerstad) [inline src/use_cases/logs.py:13]:
```suggestion
    GET selected log attributes from all logs in a given project and
    write the data to a csv file (csv file path is given by the file_path parameter).
```

**Review by KristianKjerstad — APPROVED** [review]
Good! Just 2 things you can consider changing

File: main.py (hunk 1 above)
Comment (paultrygs) [reply main.py:39]: True! Thanks 😄

**Pull request merged**